Tools/MCADefrag/MCADefrag.cpp

```diff
@@ -201,7 +201,7 @@ void cMCADefrag::cThread::ProcessFile(const AString & a_FileName)
 	// Write a copy of the Timestamps into the Out file:
 	if (Out.Write(Timestamps, sizeof(Timestamps)) != sizeof(Timestamps))
 	{
-		LOGWARNING("Cannot write Timestamps to file %s, skipping file.", OutFileName.c_str());
+		LOGWARN("Cannot write Timestamps to file %s, skipping file.", OutFileName.c_str());
```

Comment:
According to the Lua API documentation `LOGWARN` is obsolete, and we should use `LOGWARNING` instead.